unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs

```diff
@@ -20,6 +20,7 @@ public class PlacesSubSectionComponentController : IPlacesSubSectionComponentCon
     internal const int INITIAL_NUMBER_OF_ROWS = 4;
     private const int PAGE_SIZE = 12;
     private const string ONLY_POI_FILTER = "only_pois=true";
```

Comment:
Why does this const needs the `true` in it??

Reply:
Also, did you consider creating a shared enum for all this const? There is a duplication of string ids that could lead to unwanted errors